lib/rules/function-calc-no-unspaced-operator/index.js

```diff
@@ -143,12 +138,7 @@ const rule = (primary, _secondaryOptions, context) => {
 					if (currentNode.type === 'function') {
 						if (context.fix) {
 							needsFix = true;
-							nodes.splice(operatorIndex, 0, {
-								type: 'space',
-								value: ' ',
-								sourceIndex: 0,
-								sourceEndIndex: 1,
-							});
+							currentNode.value = isBeforeOp ? `${currentNode.value} ` : ` ${currentNode.value}`;
```

Comment:
Oh, dropping `*` and `/` may make the mutation unnecessary. I'll try removing it.

Reply:
I've reconsidered. Test cases may not be enough. But, just in case, I leave it.